### DISCUSSION TITLE
unstable_createNodeManifest() warning for collection routes

### DISCUSSION THREAD

**Discussion by khendrikse** [gatsbyjs/gatsby#35900] [Help]
Hi! I am using [collection routes](https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/#collection-routes) and while running `gatsby develop` I get a message like 

```
warn Plugin gatsby-source-sanity called unstable_createNodeManifest() for node id "-fakenodeid" with a manifest id of
"fakeId-2022-06-03T08:31:19.000Z" but Gatsby didn't find an ownerNodeId for the page at fake/path
Using the first page that was found with the node manifest id set in pageContext.id in createPage().
This may result in an inaccurate node manifest (for previews or other purposes). See https://www.gatsbyjs.com/docs/conceptual/content-sync for more
info.
```

I thought that for collection routes from the file-system-route-api, this is automatically done? Seeing [what is said here](https://www.gatsbyjs.com/docs/conceptual/content-sync/#node-to-page-mapping-hierarchy).

I wonder if anyone knows something I could take a look at.

**Comment by 1-800-jono** [comment]
I'm having the same problem. Same warning message.

**Comment by harrynorthover** [comment]
Yep, same here.

**Comment by kingluddite** [comment]
me too. any way to delete these nodes so i can get rid of these errors?
